gix/src/repository/blame.rs

```diff
@@ -0,0 +1,43 @@
+#[cfg(feature = "blame")]
+use gix_hash::ObjectId;
+#[cfg(feature = "blame")]
+use gix_ref::bstr::BStr;
+
+use crate::{
+    repository::{commit_graph_if_enabled, diff_resource_cache},
+    Repository,
+};
+
+/// The error returned by [`Repository::blame_file()`].
+#[derive(Debug, thiserror::Error)]
+#[allow(missing_docs)]
+pub enum Error {
+    #[error(transparent)]
+    CommitGraphIfEnabled(#[from] commit_graph_if_enabled::Error),
+    #[error(transparent)]
+    DiffResourceCache(#[from] diff_resource_cache::Error),
+    #[error(transparent)]
+    Blame(#[from] gix_blame::Error),
+}
+
+impl Repository {
+    /// Produce a list of consecutive [`gix_blame::BlameEntry`] instances. Each `BlameEntry`
+    /// corresponds to a hunk of consecutive lines of the file at `suspect:<file_path>` that got
+    /// introduced by a specific commit.
+    ///
+    /// For details, see the documentation of [`gix_blame::file::function::file()`].
```

Comment:
Shouldn't the linke be `gix_blame::file()`?

Reply:
You’re right, that also fixes the only error on CI. :-)